src/main/scala/devices/spi/TLSPI.scala

```diff
@@ -42,7 +43,8 @@ case class SPIParams(
     frameBits: Int = 8,
     delayBits: Int = 8,
     divisorBits: Int = 12,
-    sampleDelay: Int = 2)
+    sampleDelay: Int = 2,
```

Comment:
is this a width, or initial value? If initial value, should we use that in the RegInit above?

Reply:
Initial value - this is not my change though.

Reply:
ah yes, I see this was always how it was